CreatubblesAPIClient/Sources/Model/Notification.swift

```diff
@@ -91,4 +91,13 @@ open class Notification: NSObject, Identifiable
         bubbleRelationship = MappingUtils.relationshipFromMapper(mapper.bubbleRelationship)
         bubble = MappingUtils.objectFromMapper(dataMapper, relationship: bubbleRelationship, type: Bubble.self)
     }
+    
+    var isValid: Bool
```

Comment:
Logic of your statement is little bit different than original one (notification with `creationEntities` set and `creation` equal to nil can be still valid).
Also `!(creationEntities?.isEmpty)` returns `Bool?` which have to be handle different in code:
```
!(creationEntities?.isEmpty ?? true)
//or
creationEntities?.isEmpty == false
```